src/Integration.Vsix/CFamily/CFamilyIssueToAnalysisIssueConverter.cs

```diff
@@ -241,11 +231,10 @@ private string CalculateLineHash(MessagePart cFamilyIssueLocation, IReadOnlyDict
             return textSnapshot == null ? null : lineHashCalculator.Calculate(textSnapshot, cFamilyIssueLocation.Line);
         }
 
-        private AnalysisIssueLocation ToAnalysisIssueLocation(MessagePart cFamilyIssueLocation, IReadOnlyDictionary<string, ITextDocument> fileContents)
-        {
-            return new AnalysisIssueLocation
+        private AnalysisIssueLocation ToAnalysisIssueLocation(MessagePart cFamilyIssueLocation, IReadOnlyDictionary<string, ITextDocument> fileContents) =>
+            new AnalysisIssueLocation
             (
-                filePath: Path.GetFullPath(cFamilyIssueLocation.Filename),
+                filePath: Path.IsPathRooted(cFamilyIssueLocation.Filename) ? Path.GetFullPath(cFamilyIssueLocation.Filename) : cFamilyIssueLocation.Filename,
```

Comment:
We used to handle non-rooted paths for the primary location but not secondary (I don't know why). Now we handle it for both.